src/core_modules/capture-core/components/ListView/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+export { default as EventList } from './ListWrapper/EventsListWrapper.container';
+export { actionTypes as paginationActionTypes } from './Pagination';
```

Comment:
Looks like this is not used (?)

Reply:
Updated imports to use the export from here, but the code doesn't really logically fit together. Will addess this in the upcoming PRs in this chain.